### PR TITLE
AIMS-456 Add form and start of next page

### PR DESCRIPTION
**Pull request by afstanton**

### CODE REVIEW

File: app/controllers/shelves_controller.rb [hunk 1]
```diff
@@ -103,4 +103,20 @@ def dissociate
...
+  def check_trays_new
+    @shelf = Shelf.new
+  end
+
+  def check_trays
+    begin
+      @shelf = GetShelfFromBarcode.call(params[:shelf][:barcode])
+      @scanned = []
+    rescue StandardError => e
+      notify_airbrake(e)
```
Comment (rfox2) [inline app/controllers/shelves_controller.rb]: We're no longer using airbrake. Use the following for most exception handling:

```suggestion
      Raven.capture_exception(e)
```

There are other examples in the app of how we can use Raven to log errors. Take a look at the other examples and there is online documentation at Sentry.io

File: app/views/shelves/check_trays_new.html.haml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.scan
+  = form_for @shelf, url: check_trays_path(@shelf) do |f|
+    - if @shelf.errors.any?
+      #error_explanation
+        %h2= "#{pluralize(@shelf.errors.count, "error")} prohibited this shelf from being saved:"
```
[the remark this answers is not in the document]
Comment (afstanton) [reply app/views/shelves/check_trays_new.html.haml]: Good point. I started this form by copying from the other QC form (for trays) and just changed verbiage. I can certainly change this. I'd hope there wouldn't be any ActiveRecord errors here, and we shouldn't be "saving" a shelf here - though I can see creating one, if someone scanned an empty shelf that had never been scanned before, for some reason.

What should the error message say? (Also, should the corresponding one for trays be changed as well?)

[the remark this answers is not in the document]
Comment (rfox2) [reply app/views/shelves/check_trays_new.html.haml]: How about "...prohibited this operation:" ?

[the remark this answers is not in the document]
Comment (afstanton) [reply app/views/shelves/check_trays_new.html.haml]: Sounds good to me!

**Pull request merged**